Solution-Week1/script_2_sumof_digits.py

```diff
@@ -0,0 +1,10 @@
+# Write A Script To Print Sum OF Its Digit
+def sum_of_digit(num):
+    sum=0
+    while(num>0):
+        d=num%10
+        sum=sum+d
+        num=num//10

```

Comment:
why use // operator?

Reply:
Here, condition is evaluated and if num is greater than 0 then it executed the statement of  inside 'while' loop.
once loop has been iterate, interpreter extract each digit from right to left.
for extract each digit from right to left ,i use floor division.

The result of floor division(//) will be integer.
And the result of float division(/) will be float value.

 here if  i use float division operator instead of floor division ,(as float division produce result in float value because of this)
then the interpreter extract digit incorrectly due to this result will also incorrect .
that is why here i use // operator.